components/SSI/DIDxWallet/index.tsx

```diff
@@ -23,18 +18,19 @@ import { $net } from 'src/store/wallet-network';
 import { $contract } from 'src/store/contract';
 */
 
-function Component() {
+interface LayoutProps {
+  children: ReactNode;
+}
+
+function Component(props: LayoutProps) {
+  const { children } = props;
   const user = useStore($user);
   const username = user?.name;
   const domain = user?.domain;
   const arConnect = useStore($arconnect);
+  const walletInterface = useStore($walletInterface);
```

Comment:
Instead of using an interface, it'd be better to manage it with navigation.

Reply:
Yes, actually we already manage it with navigation. The wallet interface only helped us to show the back button